tests/resources/functional/snowflake/dml/delete/test_delete_with_not_in.sql

```diff
@@ -0,0 +1,15 @@
+-- snowflake sql:
+DELETE FROM test_tbl
+WHERE (version, type) NOT IN (
+SELECT version1 , type2
```

Comment:
Can you add more test case to check the behaviour., when select column names are expressions.